src/handlers/movies/getMovies.ts

```diff
@@ -0,0 +1,79 @@
+import { getAll as getAllMovies } from '../../models/Movie';
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import getAll from '../../models/Genre/getAll';
+
+export const handler: APIGatewayProxyHandler = async (event) => {
+	try {
+		const queryParams: {
```

Comment:
queryParams refers to a particular feature of HTTP protocol. Instead of calling this the same, you should rename to searchFilters or whatever you want to pass this as later